docs/Push-To-Try.md

```diff
@@ -0,0 +1,53 @@
+Prerequisites
+============
+
+* gecko-dev git repo is checkout out and updated
+* mozilla-central mercurial repo is checked out and setup
+* moz-git-tools git repo is checked out and installed on your system
+* Make sure .hg/hgrc file in mozilla-central on your local has these additions added under the "default=":
+    * Add (without quotes) "default-push = ssh://hg.mozilla.org/mozilla-central”
+    * Add (without quotes) "try = ssh://hg.mozilla.org/try”
+* At least Level 1 commit access, if you have not requested this:
+  https://www.mozilla.org/en-US/about/governance/policies/commit/
+
+
+Push To Try
+============
+
+Commit your local changes
+On Local Loop Branch:
+```shell
+$ git commit -m “Bug 123456789 - whatever message you want” .
+```
+
+Update fx-team on gecko-dev to latest, remove old branch if exists and create new branch for your changes
+On gecko-dev/fx-team:
+```shell
+$ git pull
+$ git branch -D testBranch
+$ git checkout -b testBranch
+```
+
+Build on local and export changes to gecko-dev testBranch
+On Local Loop Branch:
+```shell
+$ make build
+$ make git-export
+```
+
+Update mozilla-central to latest
+On mozilla-central:
+```shell
+$ hg pull
+$ hg update -r default
+```
+
+Create commit in testBranch on gecko-dev. Determine which commands to use for try server push and then execute try server push.
+
+* Use http://trychooser.pub.build.mozilla.org/ to get command line for try server push
+
+On gecko-dev/testBranch:
+```shell
+$ git commit -m “Test changes for Try server” .
+$ git-push-to-try -t -r HEAD ../firefox "-b do -p linux64 -u mochitest-bc,mochitest-e10s-bc -t none"
```

Comment:
You're using `mozilla-central` as the example, so you need to change the `../firefox` here.